test/src/e2e_vm_tests/test_programs/should_pass/stdlib/fixed_point_test/fixed_point_test/Cargo.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "fixed_point_test"
+version = "0.1.0"
+authors = ["Rostyslav Tyshko"]
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+fuels = "0.14"
+fuels-abigen-macro = "0.14"
```

```suggestion
fuels = "0.15"
fuels-abigen-macro = "0.15"
```

Unless the rest of the projects require an older version try to use the latest release if possible. [Version 16](https://github.com/FuelLabs/fuels-rs) is out within the last 24h. I have not used it / tested it so I suggest you try at least version 15